src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -204,7 +233,8 @@ s32 func_80AF281C(EnRu2* this, PlayState* play, u16 cueId, s32 cueChannel) {
     return false;
 }
 
-void func_80AF2868(EnRu2* this, PlayState* play, u32 cueChannel) {
+/* Checks cutscene data and, if applicable, configures Ruto's position accordingly. */
```

Comment:
The prevailing style seems to be

```c
/**
 * Checks cutscene data and, if applicable, configures Ruto's position accordingly.
 */
```
i.e. like Doxygen/Javadoc even though we don't actually run that. I'm not a stickler for this but others might be

Reply:
For function level descriptions yeah, would appreciate if you could change them to the described style (including in the review comment above this one)